cmd/get/apps/command.go

```diff
@@ -65,6 +66,9 @@ func New(config Config) (*cobra.Command, error) {
 	f := &flag{}
 
 	r := &runner{
+		commonConfig: &commonconfig.CommonConfig{
```

Comment:
We use a `commonconfig` object in each runner since it is already used in various commands.